packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -270,11 +271,20 @@
   "API_Enable_CORS": "Enable CORS",
   "API_Enable_Direct_Message_History_EndPoint": "Enable Direct Message History Endpoint",
   "API_Enable_Direct_Message_History_EndPoint_Description": "This enables the `/api/v1/im.history.others` which allows the viewing of direct messages sent by other users that the caller is not part of.",
+  "API_Enable_Personal_Access_Tokens": "Enable Personal Access Tokens to REST API",
+  "API_Enable_Personal_Access_Tokens_Description": "Enable personal access tokens for use with the REST API",
   "API_Enable_Shields": "Enable Shields",
   "API_Enable_Shields_Description": "Enable shields available at `/api/v1/shield.svg`",
   "API_GitHub_Enterprise_URL": "Server URL",
   "API_GitHub_Enterprise_URL_Description": "Example: http://domain.com (excluding trailing slash)",
   "API_Gitlab_URL": "GitLab URL",
+  "API_Personal_Access_Token_Name": "Personal Access Token Name",
+  "API_Personal_Access_Tokens_To_REST_API": "Personal access tokens to REST API",
+  "API_Personal_Access_Tokens_Remove_Modal": "Are you sure you wish to remove this personal access token?",
+  "API_Personal_Access_Token_Generated": "Personal Access Token successfully generated",
+  "API_Personal_Access_Token_Generated_Text_Token_s_UserId_s": "Please save your token carefully as you will no longer be able to view it afterwards. <br/>Token: <strong>%s</strong><br/>Your user Id: <strong>%s</strong>",
```

Comment:
please use named variables instead of `%s`, in this case, something like (look at `__token__` and `__user_id__`):
`  "API_Personal_Access_Token_Generated_Text_Token_s_UserId_s": "Please save your token carefully as you will no longer be able to view it afterwards. <br/>Token: <strong>__token__</strong><br/>Your user Id: <strong>__user_id__</strong>",`